docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template

```diff
@@ -66,59 +66,56 @@ plt.show()
 ```python
 import optuna
 import torch
-from gluonts.torch.model.deepar import DeepAREstimator
-from gluonts.mx import Trainer
+from gluonts.dataset.split.splitter import split
 from gluonts.evaluation import Evaluator
+from gluonts.mx import Trainer
+from gluonts.torch.model.deepar import DeepAREstimator
```

Comment:
This looks wrong -- `torch` and `mx` shouldn't mix`.

Looking at the docs, `Trainer` seems unused?

Reply:
Will do.